app/assets/javascripts/listings/components/process-section.html.slim

```diff
@@ -69,11 +69,7 @@ accordion-heading
     p.t-small.c-steel ng-if="::!$ctrl.parent.lotteryDateVenueAvailable($ctrl.parent.listing)"
       | Lottery date, time and venue to be scheduled.
 
-  .content-group
-    ng-include src="'listings/templates/listing/_what_to_expect.html'"
-
-  .content-group ng-if="::$ctrl.parent.listing.Leasing_Agent_Phone || $ctrl.parent.listing.Leasing_Agent_Email"
-    ng-include src="'listings/templates/listing/_contact_leasing_agent.html'"
+  ng-include src="'listings/templates/listing/_additional_info.html'"
```

Comment:
one nitpick. We have an additional info section in the listing already so this naming might be a bit confusing. 

Maybe it could be additional_sidebar_info?